mautic/mautic
# Fix issue with pre-populating the value of a form field from the URL query parameters

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [yes]
| New feature/enhancement? (use the a.x branch)      | [no]
| Deprecations?                          | [no]
| BC breaks? (use the c.x branch)        | [no]
| Automated tests included?              | [no] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
This PR proposes a solution with pre-filled field values from URL request parameters.


#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create new form or use the existing one
3. Add the form to a landing page
4. Pass values as the URL query parameters to prefil it
5. Inject the form dynamically or copy the entire output into the external page
6. Pass values as the URL query parameters to prefil it




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11077"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
